app/controllers/catalog_controller.rb

```diff
@@ -16,7 +17,30 @@ def self.modified_field
     'system_modified_dtsi'
   end
 
+  # CatalogController-scope behavior and configuration for BlacklightIiifSearch
+  include BlacklightIiifSearch::Controller
+
   configure_blacklight do |config|
+
+    # IiifPrint facet fields
+    # additional IiifPrint fields not displayed in the facet list,
+    # but below definitions give labels to filters for linked metadata
+    config.add_facet_field 'first_page_bsi', label: 'First page', if: false
+
+
+    # IiifPrint index fields
+    config.add_index_field 'all_text_tsimv', highlight: true, helper_method: :render_ocr_snippets
+
+
+    # configuration for Blacklight IIIF Content Search
+    config.iiif_search = {
+      full_text_field: 'all_text_tsimv',
+      object_relation_field: 'is_page_of_s',
```

Comment:
@laritakr 

Can we change this to is_page_of_ssim to match LV && to support multi value: https://github.com/scientist-softserv/louisville-hyku/blob/main/app/controllers/catalog_controller.rb#L35

Reply:
This is filled in by `blacklight_iiif_search_generator.rb` during install in iiif_print. It can be changed, but ideally I'd think we probably would want a config file variable rather than leaving it hardcoded in the generator file.

Reply:
Just a note: I did not change ANYTHING in this pull request. It all came directly from the iiif_print install generator.